pallets/avn-anchor/src/lib.rs

```diff
@@ -139,6 +173,10 @@ pub mod pallet {
         ValueQuery,
     >;
 
+    #[pallet::storage]
+    #[pallet::getter(fn fee_token)]
+    pub type FeeToken<T: Config> = StorageValue<_, T::Token, ValueQuery>;
```

Comment:
If we do it this way, we have to have a genesis config and a state migration to set it. IMO the previous way of passing it in the runtime was better.